src/layout/List/List.tsx

```diff
@@ -22,14 +31,55 @@ export const List: FC<ListProps & ListRef> = forwardRef<
   ListProps
 >(({ className, children, theme: customTheme, ...rest }, ref) => {
   const theme: ListTheme = useComponentTheme('list', customTheme);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
Im not sure I agree this should be in List ( or at least not in the base list component ).

Reply:
Why not in the List? It is have the same functionality as menu and user should have ability to navigate it in the same way/

Reply:
Also, it works only for items with tab index (somplly say which have onClick callback)